### PR TITLE
[zh] Sync concepts pages for system Logs

### PR DESCRIPTION
**Pull request by ydFu**
* [zh] Umbrella issue: pages out of sync in concepts section(#26177 system Logs)

`
[x] content/zh/docs/concepts/cluster-administration/system-logs.md
`

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/system-logs.md [hunk 1]
```diff
@@ -153,6 +153,54 @@ List of components currently supporting JSON format:
...
+{{<warning >}}
+日志清理可能会导致大量的计算开销，因此不应启用在生产环境中。
+{{< /warning >}}
+
+<!--
+The `--experimental-logging-sanitization` flag enables the klog sanitization filter.
+If enabled all log arguments are inspected for fields tagged as sensitive data (e.g. passwords, keys, tokens) and logging of these fields will be prevented.
+-->
+
+`--experimental-logging-sanitization` 参数启用了 klog 清理过滤器。
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/system-logs.md:179]:
```suggestion
`--experimental-logging-sanitization` 参可用来启用 klog 清理过滤器。
```

File: content/zh/docs/concepts/cluster-administration/system-logs.md [hunk 2]
```diff
@@ -153,6 +153,54 @@ List of components currently supporting JSON format:
...
+* kube-apiserver
+* kube-scheduler
+* kubelet
+
+{{< note >}}
+The Log sanitization filter does not prevent user workload logs from leaking sensitive data.
+{{< /note >}}
+-->
+
+当前支持日志清理的组件列表：
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/system-logs.md:194]: 194 之后添加一个空行

File: content/zh/docs/concepts/cluster-administration/system-logs.md (hunk 1 above)
Comment (ydFu) [reply content/zh/docs/concepts/cluster-administration/system-logs.md]: thank you for suggesting me.

File: content/zh/docs/concepts/cluster-administration/system-logs.md (hunk 2 above)
Comment (ydFu) [reply content/zh/docs/concepts/cluster-administration/system-logs.md:194]: thank you for suggesting me.

**Pull request merged**